Xamarin.Forms.Core.UnitTests/ImageSourceTests.cs

```diff
@@ -56,6 +56,15 @@ public void TestImplicitFileConversion ()
 		}
 
 		[Test]
+		public void TestImplicitStringConversionWhenNull()
```

Comment:
This test (where the input is `null`) seemed to be completely missing, so I added it. Note that I am only asserting the existing behavior here - I don't actually _agree_ with the behavior, but it is what it is.